### DISCUSSION TITLE
How to describe variable under cursor, to see it all fields?

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [neoclide/coc.nvim#3931] [Q&A] [answered]
I have `nnoremap <silent> <leader>h :call CocActionAsync('doHover')<cr>` but sometimes it returns only type name. I use TypeScript.

And related question - is it possible to see function definition in the popup, full code?

**Comment by fannheyward** [comment, marked as answer]
Change `doHover` to `definitionHover`, check `:h definitionHover`.
